examples/images_contours_and_fields/colormap_interactive_adjustment.py

```diff
@@ -3,90 +3,77 @@
 Interactive Adjustment of Colormap Range
 ========================================
 
-Demonstration of using colorbar, picker, and event functionality to make an
-interactively adjustable colorbar widget.
+Demonstration of how a colorbar can be used to interactively adjust the
+range of colormapping on an image. To use the interactive feature, you must
+be in either zoom or pan mode and click inside the colorbar.
```

Comment:
```suggestion
be in either zoom mode (magnifying glas toolbar button) or pan mode (4-way arrow toolbar button)  and click inside the colorbar.
```
I suspect not every user knows what "zoom or pan mode" means.